doc/classes/Path2D.xml

```diff
@@ -13,5 +13,14 @@
 		<member name="curve" type="Curve2D" setter="set_curve" getter="get_curve">
 			A [Curve2D] describing the path.
 		</member>
+		<member name="debug_custom_color" type="Color" setter="set_debug_custom_color" getter="get_debug_custom_color" default="Color(1, 1, 1, 1)">
+			The custom color used to draw the path debug when [member debug_custom_enabled] is enabled.
+		</member>
+		<member name="debug_custom_enabled" type="bool" setter="set_debug_custom_enabled" getter="get_debug_custom_enabled" default="false">
+			If [code]true[/code], the debug custom properties override the global defaults.
+		</member>
+		<member name="debug_enabled" type="bool" setter="set_debug_enabled" getter="get_debug_enabled" default="true">
+			Used at runtime to toggle debug when "Visible Paths" is enabled in the Editor Debug menu. Inside the Editor debug is always enabled.
```

Comment:
This description only works with context. Without reading the property's name and type, you'd not understand it.
I think this is related to `debug_paths_hint`?
```suggestion
			If [code]true[/code], allows this path to be displayed for debugging purposes. Used when "Visible Paths" is enabled in the Editor Debug menu. In the editor, this property has no effect. See also [member SceneTree.debug_paths_hint].
```